sdk/dotnet/Pulumi/Core/InputOneOf.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright 2016-2018, Pulumi Corporation
+
+using OneOf;
+
+namespace Pulumi.Core
```

Comment:
Why is this in `Pulumi.Core` instead of `Pulumi` (like `InputMap` and `InputList`)?  It appears to be the only thing defined in this namespace?

Reply:
Yup.  htat's a mistake.